storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -122,6 +122,18 @@ public final class RemoteLogManagerConfig {
             "is full, fetch requests are served with an error.";
     public static final int DEFAULT_REMOTE_LOG_READER_MAX_PENDING_TASKS = 100;
 
+    public static final String LOG_LOCAL_RETENTION_MS_PROP = "log.local.retention.ms";
+    public static final String LOG_LOCAL_RETENTION_MS_DOC = "The number of milliseconds to keep the local log segment before it gets deleted. " +
```

Comment:
should we perhaps add here that this configuration is only used when remote storage is enabled at a system level?

(same for retention bytes)